src/activity_tracker/activity_tracker.c

```diff
@@ -0,0 +1,169 @@
+            imageRomPlayTime = TTF_RenderUTF8_Blended(font40, play_time_total_formatted, color_white);
+            imageRomName = TTF_RenderUTF8_Blended(fontRomName25, play_activities->play_activity[index]->rom->name, color_white);
+            snprintf(details, 100, "count: %d average: %d:%02d", play_activities->play_activity[index]->play_count, play_activities->play_activity[index]->play_time_average/3600, (play_activities->play_activity[index]->play_time_average-((play_activities->play_activity[index]->play_time_average/3600)*3600))/60);
+            imageRomDetails = TTF_RenderUTF8_Blended(font18, details, color_white);
+            SDL_Rect rectPosition = {16, 80 + 90 * i, 50, 39};
+            SDL_Rect rectRomImage = {70, 70 + 90 * i, 80, 80};
+            SDL_Rect rectRomPlayTime = {170, 55 + 90 * i, 100, 56};
+            SDL_Rect rectRomNames = {170, 100 + 90 * i, 400, 40};
+            SDL_Rect rectRomDetails = {170, 130 + 90 * i, 400, 40};
+            SDL_BlitSurface(imageRomPosition, NULL, screen, &rectPosition);
+            SDL_BlitSurface(imageRomImage, NULL, screen, &rectRomImage);
+            SDL_BlitSurface(imageRomPlayTime, NULL, screen, &rectRomPlayTime);
+            SDL_BlitSurface(imageRomName, NULL, screen, &rectRomNames);
+            SDL_BlitSurface(imageRomDetails, NULL, screen, &rectRomDetails);
+        }
+    }
+    h = play_time_total / 3600;
+    m = (play_time_total - (3600 * h)) / 60;
+    sprintf(play_time_total_formatted, "%d:%02d", h, m);
+    imageMileage = TTF_RenderUTF8_Blended(font30, play_time_total_formatted, color_white);
+    SDL_BlitSurface(imagePages, NULL, screen, &rectPages);
+    SDL_BlitSurface(imageMileage, NULL, screen, &rectMileage);
+    SDL_BlitSurface(screen, NULL, video, NULL);
+    SDL_Flip(video);
+    bool changed;
+    KeyState keystate[320] = {(KeyState)0};
+    while (!quit) {
+        msleep(20);
+        changed = false;
+        if (updateKeystate(keystate, &quit, true, NULL)) {
+            if (keystate[SW_BTN_B] == PRESSED)
+                quit = true;
+            if (keystate[SW_BTN_RIGHT] >= PRESSED) {
+                if (nCurrentPage < nPages - 1) {
+                    nCurrentPage++;
+                    changed = true;
+                }
+            }
+            if (keystate[SW_BTN_LEFT] >= PRESSED) {
+                if (nCurrentPage > 0) {
+                    nCurrentPage--;
+                    changed = true;
+                }
+            }
+        }
+        if (!changed)
+            continue;
+        // Page update
+        SDL_BlitSurface(imageBackground, NULL, screen, NULL);
+        sprintf(cPages, "%d/%d", nCurrentPage + 1, nPages);
+        imagePages = TTF_RenderUTF8_Blended(font30, cPages, color_white);
+        SDL_BlitSurface(imagePages, NULL, screen, &rectPages);
+        SDL_BlitSurface(imageMileage, NULL, screen, &rectMileage);
+        for (int i = 0; i < 4; i++) {
```

Comment:
Move this whole thing into its own function. There's no reason to duplicate this code.